test/rules/no-duplicate-super/test.ts.lint

```diff
@@ -259,5 +259,14 @@ declare const n: number;
     }
 }
 
+// With ternary operator
+{
+    class {
```

Comment:
btw. this is not a valid class declaration. Class declarations need a name.

Reply:
I thought about it but that's how it's done in all the other test cases for that rule. I've figured out there's some reason for it and followed it. I'll fix it in all test cases with updated PR.